### PR TITLE
[Hub Generated] Review request for Microsoft.CostManagement to add version preview/2020-12-01-preview

### PR DESCRIPTION
**Pull request by Zzegfried**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/melmullen/azure-rest-api-specs/dev-cost-management-Microsoft.CostManagement-2020-12-01-preview...master/).

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2020-12-01-preview/costmanagement.exports.json [hunk 1]
```diff
@@ -0,0 +1,880 @@
...
+        "container": {
+          "description": "The name of the container where exports will be uploaded. If the container does not exist it will be created.",
+          "type": "string"
+        },
+        "rootFolderPath": {
+          "description": "The name of the directory where exports will be uploaded.",
+          "type": "string"
+        },
+        "sasToken": {
+          "description": "A SAS token for the storage account. For a restricted set of Azure customers this together with storageAccount can be specified instead of resourceId.",
```
Comment (paornela) [inline specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2020-12-01-preview/costmanagement.exports.json:644]: Is there a reason for not specifying which Azure customers can use this? I foresee CRIs with customer questions coming about this

Comment (melmullen) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2020-12-01-preview/costmanagement.exports.json:644]: This is already covered out in the associated description of the parent element.

File: specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2020-12-01-preview/examples/ExportCreateOrUpdateByBillingAccount.json [hunk 2]
```diff
@@ -0,0 +1,128 @@
+{
+  "parameters": {
+    "api-version": "2020-06-01",
```
Comment (absyed) [inline specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2020-12-01-preview/examples/ExportCreateOrUpdateByBillingAccount.json:3]: Api version needs to be 2020-12-01-preview for here and rest of the places

Comment (melmullen) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2020-12-01-preview/examples/ExportCreateOrUpdateByBillingAccount.json]: Duh! Thank you.

**Review by paornela — APPROVED** [review]

**Review by absyed — APPROVED** [review]

File: specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2020-12-01-preview/costmanagement.exports.json [hunk 3]
```diff
@@ -0,0 +1,880 @@
...
+        },
+        "container": {
+          "description": "The name of the container where exports will be uploaded. If the container does not exist it will be created.",
+          "type": "string"
+        },
+        "rootFolderPath": {
+          "description": "The name of the directory where exports will be uploaded.",
+          "type": "string"
+        },
+        "sasToken": {
```
Comment (pilor) [inline specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2020-12-01-preview/costmanagement.exports.json:643]: Please follow https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi013-secrets-must-be-annotated-with-x-ms-secret-and-x-ms-mutability

Comment (melmullen) [reply specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2020-12-01-preview/costmanagement.exports.json:643]: Updated as suggested. Thank you.

**Comment by ArcturusZhang** [comment]
@msyyc could you take a look at the breaking changes reported in python pipeline?

**Comment by msyyc** [comment]
For python, the breaking change is ok

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**